Utilities/DataLoading/EdFi.LoadTools/BulkLoadClient/LoadProcess.cs

```diff
@@ -135,82 +125,6 @@ private static void LogConfiguration(Configuration configuration)
             _log.Info($"Dependencies Url:         {configuration.DependenciesUrl}");
         }
 
-        private static void ConfigureCompositionRoot(Container container, Configuration configuration)
```

Comment:
This was isolated into this class to help Admin App consume the load tools in a class library fashion, as opposed to needing to call it as a CLI.  This will completely break Admin App with these changes.

Reply:
Reverted to use the static class